docs/topics/classes.md

```diff
@@ -29,8 +29,8 @@ If the primary constructor does not have any annotations or visibility modifiers
 class Person(firstName: String) { /*...*/ }
 ```
 
-The primary constructor cannot contain any code. Initialization code can be placed in _initializer blocks_ prefixed with
-the `init` keyword.
+The primary constructor initializes a class instance and its properties. Initialization code can be placed in _initializer
```

Comment:
Why have we decided to omit "cannot containing any code"?
Is it insignificant?

Reply:
This sentence was causing confusion for some readers. They thought initializer blocks were code in the primary constructor even though the primary constructor is only defined in the class header. So I thought it would be cleaner to remove the sentence entirely.

Reply:
Well, maybe we should write explicitly about "primary constructor is only defined in the class header"?
What do you think?

Reply:
Anyway, feel free to merge the PR :)

Reply:
Thanks for your suggestion! I updated this paragraph with a couple more sentences to make it clear that the primary constructor is declared in the class header which can't contain any runnable code. I also explain that to run some code as part of object creation, users can use initialization blocks. Hopefully everything is covered now :)